src/BattleCalculator-Freewar/BattleCalculator-Freewar.user.js

```diff
@@ -75,10 +105,10 @@ function processElement(cellElement) {
 			$(npcNameElement).addClass('processedNPC knownNPC');
 		} else if (lifeLoss == -1) {
 			// Player looses
-			$(npcFastAttackElement).css('color', '#F00F0F');
-			$(npcFastAttackElement).append(' (defeat)');
 			$(npcFastAttackElement).removeAttr('href');
 			$(npcFastAttackElement).removeAttr('onclick');
+            $(npcFastAttackElement).off('click');
+            $(npcFastAttackElement).text('Defeat').css({color: 'red',fontWeight: 'bold'});
```

Comment:
NIT, u used spaces instead of tabs everywhere. looks kinda odd

Reply:
fixxed with https://beautifier.io/... muss ich mir angewöhnen schöneren code zu schreiben / editieren :D